app/connectors/ArrivalMovementConnector.scala

```diff
@@ -37,12 +37,26 @@ class ArrivalMovementConnector @Inject() (config: FrontendAppConfig, http: HttpC
 
   private val channel: String = "web"
 
-  def getArrivals()(implicit hc: HeaderCarrier): Future[Option[Arrivals]] = {
+  private def makeQueryParam(param: String, valueParam: Option[String]): Seq[(String, String)] =
+    valueParam.fold(Seq.empty[(String, String)])(
+      value => Seq((param, value))
+    )
+
+  def getArrivals(
+    mrn: Option[String] = None,
+    page: Option[String] = None,
+    pageSize: Option[String] = None
```

Comment:
Good idea to allow the call site to provide this. We can make it so that it's easy for us to experiment with and configure

Reply:
I didn't feel right either hard-coding it or binding it into app config at that level. :)